mmtrack/datasets/vot_dataset.py

```diff
@@ -1,27 +1,32 @@
+import glob
 import os.path as osp
+import time
 
+import mmcv
 import numpy as np
 from mmcv.utils import print_log
 from mmdet.datasets import DATASETS
 
 from mmtrack.core.evaluation import eval_sot_accuracy_robustness, eval_sot_eao
-from .sot_test_dataset import SOTTestDataset
+from .base_sot_dataset import BaseSOTDataset
 
 
 @DATASETS.register_module()
-class VOTDataset(SOTTestDataset):
-    """VOT dataset for the testing of single object tracking.
+class VOTDataset(BaseSOTDataset):
+    """VOT dataset of single object tracking.
 
-    The dataset doesn't support training mode.
-
-    Note: The vot datasets using the mask annotation, such as VOT2020, is not
-    supported now.
+    The dataset is only used to test.
     """
-    CLASSES = (0, )
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, challenge_year, *args, **kwargs):
+        """Initialization of SOT dataset class.
+
+        Args:
+            challenge_year (int): The year of VOT challenge.
```

Comment:
`challenge_year` -> `dataset_type`.
And add optional values in the docstring following [this](https://github.com/open-mmlab/mmtracking/blob/master/tools/convert_datasets/vot/vot2coco.py#L29)